src/cargo/ops/cargo_doc.rs

```diff
@@ -1,4 +1,6 @@
 use std::collections::HashSet;
+#[cfg(not(any(target_os = "windows", target_os = "macos")))]
+use std::env;
```

Comment:
Could you add this `use` to the `open_docs` method below? Should help avoid duplication of the `#[cfg]`